### PR TITLE
Ticket 721 - Pen tool widget

### PR DESCRIPTION
**Pull request by eyosef**
- `penContent.tsx` 
    - leverages React hook `useEffect()` to initialize ESRI's `SketchViewModal`
    - function, `setDrawTool()` leverages `react-redux`'s `useDispatch` to close the modal. Also allows user to draw a polygon on the map via `mapController.createPolygonSketch()`
    - leverages SVGs
    - updated content to match comps in `master` branch
- `mapController.ts` 
    - `_mapview` and `_sketchVM` are updated to have a type of any to account for specific methods that are fired in this PR.
    - `_previousSketchGraphic` tracks previous graphics
    - `initializeAndSetSketch()` creates a new instance of `SketchViewModal`. Leverages `.on('create'` logic to update the outline color and fill color when the user finishes drawing the polygon. Matches comps in `master`. Also uses `store` to dispatch to render the analysis tab in the left panel.
    - `createPolygonSketch()` allows the user to draw 'freehand' polygons on the map. Also removes the previous graphic via `_previousSketchGraphic`
- `penContent.scss` styled to be mobile-first responsive
- `modalCard.scss` has styling for the orange button to be used across modal card content, `.orange-button`
- `variables.scss` updated to include new SASS variable for `.orange-button`
- New icons: `plusIcon.svg`, `polygonIcon.svg` 

<br/>

* Fixes https://github.com/wri/gfw-mapbuilder/issues/721

### CODE REVIEW

File: src/js/components/mapWidgets/widgetContent/penContent.tsx [hunk 1]
```diff
@@ -1,9 +1,21 @@
-import React, { FunctionComponent } from 'react';
-import { useSelector } from 'react-redux';
+import React, { FunctionComponent, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
```
Comment (vaidotasp) [inline src/js/components/mapWidgets/widgetContent/penContent.tsx:3]: Can we use relative imports for most things? eg `from images/penIcon.svg` `from js/store/appState...` etc etc

File: src/js/controllers/mapController.ts [hunk 2]
```diff
@@ -12,11 +14,13 @@ interface ZoomParams {
 
 export class MapController {
   _map: Map | null;
-  _mapview: MapView | null;
+  _mapview: MapView | null | any;
```
Comment (vaidotasp) [inline src/js/controllers/mapController.ts]: why `any`? that will prevent us from utilizing esri baked in types for both `mapview` and `sketchVM`

File: src/css/modalCard.scss [hunk 3]
```diff
@@ -43,6 +43,28 @@
   height: 100%;
 }
 
+.orange-button {
```
Comment (eyosef) [inline src/css/modalCard.scss:51]: Note: hoist to app.scss or variables.scss

Comment (eyosef) [reply src/css/modalCard.scss:51]: `.orange-button` will be maintained by single source of truth (`resources.js`)

**Review by SwampGuzzler — APPROVED** [review]
Looking good, especially on `mobile`!

**Pull request merged**